pkgs/by-name/ai/aider/package.nix

```
@@ -0,0 +1,104 @@
+{
+  lib,
+  python311,
+  fetchFromGitHub,
+  git,
+  portaudio,
+}:
+
+let
+  python3 = python311.override {
+    self = python3;
+    packageOverrides = _: super: { tree-sitter = super.tree-sitter0_21; };
+  };
+in
+python3.pkgs.buildPythonApplication rec {
+  pname = "aider";
```

```suggestion
  pname = "aider-chat";
```
`aider` is a different package. Please rename it to `aider-chat`.
https://pypi.org/project/aider/
https://pypi.org/project/aider-chat/

---

Commit message and PR title should be updated to reflect `aider-chat` as well.